gr-qtgui/include/gnuradio/qtgui/CMakeLists.txt

```diff
@@ -24,16 +24,14 @@ install(
           constellationdisplayform.h
           displayform.h
           edit_box_msg.h
-          eye_sink_c.h
```

Comment:
These files were added back as wrappers, but are not in the CMakeFile.txt.

Reply:
Oh, sry I'll check that